README.md

```diff
@@ -164,5 +201,7 @@ Write a route in sinatra that will print "hello world" in the web browser at the
 
 Your answer:
 ```
-Replace this with your answer
+def "/oh_hello"
+  puts "hello world"
+end
 ```
```

Comment:
you want `return` here instead of `puts` since puts will just output to the console and returns nil